src/Libraries/CoreNodeModels/Input/BaseTypes.cs

```diff
@@ -19,7 +19,7 @@ namespace Dynamo.Nodes
 {
     [NodeName("String")]
     [NodeCategory(BuiltinNodeCategories.CORE_INPUT)]
-    [NodeDescription("Creates a string.")]
+    [NodeDescription("StringInputNodeDescription", typeof(Properties.Resources))]
```

Comment:
Thanks a lot for catching this, @holyjewsus. I had to merge `localization` branch in for it to be tested by the build machines, since this whole up-merging is an coordinated effort that is on-going for a while.

Hi @nguyen-binh-minh, @Sancerio and @zora-wang, could anyone of you fix these on the `master` branch? I might not get to this for a quick while. Thanks!